department-of-veterans-affairs/va.gov-cms
# VACMS-3494: Add non-beneficiaries to r&s tag selections.

## Description
See #3494 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/100913450-cbe29280-349f-11eb-89e2-7fcc1f14bd08.png)

## QA steps
- As an administrator, go to `/node/9544/edit`
- Visually verify that non-beneficiaries are available in the Audience dropdown
- Visually verify that when either one Beneficiary, or non-beneficiary item is selected, no more than 3 topics can be selected.

### Definition of Done

### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
